<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AirGate Giriş Sistemi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string></string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <widget class="QLabel" name="labelHomeText">
      <property name="text">
       <string>Kartınızı Okutun</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCityMain">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTimeMain">
      <property name="text">
       <string>Saat: --:--:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDateMain">
      <property name="text">
       <string>Tarih: ----/--/--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutDefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
